Umbraco-Cloud/Set-Up/Dedicated/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Move your Umbraco Cloud project to dedicated resources
+
+In this article, you can read about how you enable your Umbraco Cloud project to run on dedicated resources and what you need to be aware of before you do so.
+
+## Before you move your project to dedicated resources
+
+Before you decide to move your Umbraco Cloud project, you need to consider a few things:
+
+- Currently, Umbraco Cloud offers dedicated resources for professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.
+
+- In case you want to move your Umbraco Cloud project with a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
```

Comment:
```suggestion
- In case you want to move a project on a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
```